meetup/models.py

```diff
@@ -1,29 +1,24 @@
 from django.db import models
 from pizzaplace.models import PizzaPlace
 from django.core.validators import RegexValidator
-from meetup.services.meetup_api_lookup_agent import MeetupApiLookupAgent
+from meetup.services.meetup_api import MeetupApi
 from django.core.exceptions import ValidationError
-
+from model_utils.models import TimeStampedModel
 
 def validate_urlname(link):
   validator = RegexValidator(
     regex='meetup\.com\/\w+(-\w+)*\/$',
-    message="Does not conform to Meetup Url",
+    message="Url should be in form 'meetup.com/meetup-name/'",
     code='invalid_url')
   return validator(link)
 
 def validate_meetup_exists(link):
-  looker = MeetupApiLookupAgent(link)
-  is_real = looker.is_real_meetup()
-
-  if not is_real:
-    raise ValidationError("That's not a meetup")
-
-
+  if not MeetupApi(link).url_exists():
```

Comment:
Actually, let me be more explicit: URLs and Meetups are not concepts at the same level of abstraction. When I talk with the Meetup API, I expect to be talking about Meetups themselves. Their URLs are a secondary concern. I don't care if the Meetup's URL exists, I care if the Meetup exists. This class should allow me to ask that question.